_config.yml

```diff
@@ -904,3 +904,35 @@ oss_projects:
     link: artsy/rosalind
     repository: https://github.com/artsy/rosalind
     created: Dec 22nd, 2016
+
+#  ███████████               █████                             █████   
+# ░░███░░░░░███             ░░███                             ░░███    
+#  ░███    ░███  ██████   ███████   ██████   ██████    █████  ███████  
+#  ░██████████  ███░░███ ███░░███  ███░░███ ░░░░░███  ███░░  ░░░███░   
+#  ░███░░░░░░  ░███ ░███░███ ░███ ░███ ░░░   ███████ ░░█████   ░███    
+#  ░███        ░███ ░███░███ ░███ ░███  ███ ███░░███  ░░░░███  ░███ ███
+#  █████       ░░██████ ░░████████░░██████ ░░████████ ██████   ░░█████ 
+# ░░░░░         ░░░░░░   ░░░░░░░░  ░░░░░░   ░░░░░░░░ ░░░░░░     ░░░░░  
+                       
+# NOTE: this needs to be kept at the very end of the file.
+podcast:
+  description: A podcast exploring questions in software engineering.
+  language: en
+  copyright: Artsy, 2020
+  keywords: 
+   - engineering
+   - software engineering
+   - software development
+   - open source
+  author: Artsy Engineering
+  email: OpenSourceGroup@artsy.net
+  logo_url: https://artsy.github.io/images/podcast_logo.png
+  episodes:
+    - title:
+      date:
+      description:
+      url: 
+      file_byte_length: 
+      duration:
+      credits:
+        -
```

Comment:
The dependencies were successfully installed 🎉 It did fail to build because this first episode is blank. Excited to move forward – I'll leave it up to you, @pepopowitz!

Reply:
I opened #637 against this PR/branch, figuring we could merge it in here to get a solid CI check.